homework-solutions/js/hw-3/task-2.js

```diff
@@ -4,3 +4,20 @@
 n + nn + nnn, где n не перемножаются, а конкатенируются
 
 */
+
+let n = 4; 
+let nn = n + "" + n;
+let nnn = n + "" + n + "" + n;;
```

Comment:
Достаточно одной строки для преобразования всего выражения в строку
```suggestion
const nnn =  "" + n  + n + n;;
```